classes/module/Module.php

```diff
@@ -472,6 +477,56 @@ public function install()
         return true;
     }
 
+    protected function checkOverrideConflicts()
+    {
+        if (!is_dir($this->getLocalPath() . 'override')) {
+            return true;
+        }
+    
+        foreach (Tools::scandir($this->getLocalPath() . 'override', 'php', '', true) as $file) {
+            $module_override_path = $this->getLocalPath() . 'override/' . $file;
+            $prestashop_override_path = _PS_ROOT_DIR_ . '/override/' . $file;
+    
+            if (file_exists($prestashop_override_path)) {
+                if (!$this->compareOverrideFiles($module_override_path, $prestashop_override_path)) {
+                    $this->_errors[] = Context::getContext()->getTranslator()->trans(
+                        'The override file %1$s conflicts with an existing override in %2$s.',
+                        [$module_override_path, $prestashop_override_path],
+                        'Admin.Modules.Notification'
+                    );
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+    
+    protected function compareOverrideFiles($module_override_path, $prestashop_override_path)
```

Comment:
should be private by default, as it is only used by our code. and does not need to be overriden